File: Java_Programming/Competitive/Question6.java
// write a program to print multiplication table of a number .

class Logic 
{
    void printTable(int iNo)
    {
        int iCnt = 0 ;

        System.out.println("Table : ");
        for(iCnt = 1 ; iCnt <= 10 ; iCnt++)
        {
            System.out.println(iNo*iCnt);
        }
    }
}

class Question5
{
    public static void main(String A[])
    {
        Logic lobj = new Logic();
        lobj.printTable(11 );
    }
}

File: Python_Programming/Competitive/Question6.py
'''
        write a program which accepts one number and prints multiplication table of that number

        Input : 4
        Output : 4  8   12  16  20  24  28  32  36  40
'''

def PrintTable(No):
    for i in range(4 , 41 , 4):
        print(i,"\t")

def main():
    Value = int(input("Enter number : "))

    PrintTable(Value)

if __name__ == "__main__":
    main()